supporting_content/dimensions/listunitprice.md

```diff
@@ -0,0 +1,83 @@
+# Column: ListUnitPrice
+
+## Example provider mappings
+
+Current column mappings found in available data sets:
+
+| Provider  | Data set                     | Column                   |
+|:----------|:-----------------------------|:-------------------------|
+| AWS       | CUR;<br>AWS Price List API   | pricing/publicOnDemandCost: The total cost for the line item based on public On-Demand Instance rates. If you have SKUs with multiple On-Demand public costs, the equivalent cost for the highest tier is displayed. For example, services offering free-tiers or tiered pricing.<br>Since the pricing/publicOnDemandCost currently doesn't provide the correct price in all tier-based pricing use cases, we need to rely on the AWS Price List API. |
+| GCP       | BigQuery Pricing Data Export | Not available in BigQuery Billing Export, but can be resolved from the list_price Struct in the BigQuery Pricing Data Export |
+| Microsoft | Cost details;<br>Azure Retail Prices REST API | pay-as-you-goPrice/payGPrice: Retail price for the resource<br>Note: Identified some discrepancy between the value in Cost details and the Retail Price obtained from the Azure Retail Prices REST API, particularly in the context of CSPs. Might be related to Partner Billing UC? |
+| OCI       | List Pricing REST API        | Not available in Cost and Usage Report but can be resolved from the List Pricing REST API |
+
+## Example scenarios for current provider data
+
+Current values observed in billing data for various scenarios:
+
+| Provider  | Scenario                                 | Pattern                   |
+|:----------|:-----------------------------------------|:----------------------------|
+| AWS       | Flat-rate and tier-based list unit price | See [Current ListUnitPrice UCs and Data Samples - AWS sheet](https://docs.google.com/spreadsheets/d/1TpoXiu3aW_ENXvbONsjb6S1fqC7tJ8RcBNtFbsS9Qjw/edit#gid=726782282) for current data samples, as available in CUR files and AWS Price List API |
```

Comment:
What does `UCs` stand for? We should probably spell that out.

Reply:
Provided more information about flat-rate and usage-dependent pricing